### DISCUSSION TITLE
[New] Des pages de navigation contiennent le sommaire des sous-thèmes

### DISCUSSION THREAD

**Discussion by Ferreole** [cnumr/best-practices#89] [Bonnes pratiques] [answered]
Bonne pratique #5107   
--
4, conseillé | 2, MEO standard | 2, impact moyen
Gains :    Personne     Planète | Chaque page de navigation présente les sous-thèmes.   L’objectif :    faciliter la navigation entre les pages et les   niveaux,    faciliter l’accès aux sous-pages sur mobile,   favoriser le référencement.   La page mère contient une description brève et un lien   vers la sous-page.

Voici une proposition de bonne pratique. Pour la classification, c'est celle que je suis ds mes BP de sobriété éditoriale. A dispo pour échanger

**Comment by ACTLEM** [comment]
Note équipe projet: Renforce l'accessibilité. A voir si cela peut compléter via une nouvelle bonne pratique celle sur la fluidité du process (ici fluidité de la navigation vs fluidité dans la saisie d'un formulaire).

**Comment by Christophe-Cl** [comment]
Je ne comprends pas la BP. Un exemple ou contre-exemple ?

**Reply by Ferreole** [reply]
Par exemple : https://www.cyclop-editorial.fr/formation-contenus-web/
ca réduit le temps de recherche et facilite l'accès aux contenus

**Reply by Christophe-Cl** [reply]
J'ai le menu classique avec ses sous-menus en roll-over, aussi classiques (cf capture 1)
OU le "burger menu" en responsive, aussi classique même si ici c'est non-ergonomique (cf capture 2)

et je ne vois pas la valeur ajoutée de ce procédé en terme d'écoconception, si ce n'est réduire le nombre de clics peut-être ?

![Capture d’écran 2021-10-29 à 17 53 15](https://user-images.githubusercontent.com/92570233/139465743-1ecb32ee-ada6-4711-90ea-36359a559f76.png)
 
![Capture d’écran 2021-10-29 à 17 53 40](https://user-images.githubusercontent.com/92570233/139465764-f13c7875-c400-4320-8c0e-1e90e92e8d78.png)

**Reply by tbroyer** [reply]
Je crois que Ferréole pointait plutôt le _contenu_ de la page, qu'on retrouve effectivement dans le sous-menu "Formation" mais ici avec une description et pas un simple titre. Si j'ai bien compris, si on n'est pas sûr de la sous-rubrique qui nous intéresse, on peut donc naviguer sur cette page intermédiaire qui nous aidera à choisir. Ça fait potentiellement un chargement de page en plus, mais ça peut aussi éviter plusieurs chargements de page si on visite les pages une par une à la recherche de ce qui nous intéresse.

@Ferreole Est-ce que j'ai bien compris le sens de la proposition ?

**Comment by Ferreole** [comment]
Oui Thomas c'est exactement cela, décidément tu clarifie ma pensée ;-)

Le 01/11/2021 à 01:35, Thomas Broyer a écrit :
>
> Je crois que Ferréole pointait plutôt le /contenu/ de la page, qu'on 
> retrouve effectivement dans le sous-menu "Formation" mais ici avec une 
> description et pas un simple titre. Si j'ai bien compris, si on n'est 
> pas sûr de la sous-rubrique qui nous intéresse, on peut donc naviguer 
> sur cette page intermédiaire qui nous aidera à choisir. Ça fait 
> potentiellement un chargement de page en plus, mais ça peut aussi 
> éviter plusieurs chargements de page si on visite les pages une par 
> une à la recherche de ce qui nous intéresse.
>
> @Ferreole <https://github.com/Ferreole> Est-ce que j'ai bien compris 
> le sens de la proposition ?
>
> —
>


-- 
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Comment by ACTLEM** [comment, marked as answer]
Suite à la visio du 12/11 je me permets de mettre en résolu cette bonne pratique.
